spec/lib/gateway/stat_gateway_spec.rb

```diff
@@ -123,4 +141,37 @@
       expect(subject.signups[:sms_today]).to eq(1)
     end
   end
+
+  context 'given email signups made on different dates' do
+    before do
+      User.create(
+        username: "Email old",
+        created_at: Date.today - 1,
+        contact: 'foo@bar.com',
+        sponsor: 'foo@bar.com'
+        )
+
+      User.create(
+        username: "Email new",
+        contact: 'foo@baz.com',
+        sponsor: 'foo@baz.com'
+        )
+    end
+
+    it 'counts them against total singups' do
+      expect(subject.signups[:total]).to eq(2)
+    end
+
+    it 'counts them against todays signups' do
+      expect(subject.signups[:today]).to eq(1)
+    end
+
+    it 'counts them against email total signups' do
+      expect(subject.signups[:email_total]).to eq(2)
+    end
+
+    it 'counts them against email todays signups' do
```

Comment:
This feels a little jarring.

`counts them against todays email signups` is nicer.